sql/003_items.sql

```diff
@@ -79,6 +79,11 @@ CREATE OR REPLACE FUNCTION create_item(data jsonb) RETURNS VOID AS $$
     INSERT INTO items (content) VALUES (data);
 $$ LANGUAGE SQL SET SEARCH_PATH TO pgstac,public;
 
+-- Bulk insert function which takes a sequence of items as input
+CREATE OR REPLACE FUNCTION create_items(datas jsonb) RETURNS VOID AS $$
+    INSERT INTO items (content) SELECT * FROM jsonb_array_elements(datas);
+    SELECT backfill_partitions();
```

Comment:
@bitner is this the best way to update partitions?

Reply:
Yeah, that will work just fine. I'm assuming you modeled this off the pypgstac loader code - for the insert case where we fail if we hit a duplicate id, in the case that the partitions are likely to already exist, it's faster to do like this and add the items first (any items that do not have a partition yet will go into the items_default partition) and then backfill. When the partitions don't exist yet for any data, it is going to be slower in this case than following the code in copy_ignore_duplicates and copy_upsert in pypgstac load where we bring the data into a temporary table first, pre-generate any missing partitions (getting the range from the temp table) and then load the data into the items table either using "on conflict do nothing" to ignore dupes or "on conflict do update..." to upsert.

Reply:
pg_partman was dropped, so we should remove this line right?